packages/svelte/src/compiler/phases/2-analyze/validation.js

```diff
@@ -568,6 +568,17 @@ const validation = {
 			}
 		}
 
+		// can't add form to interactive elements because those are also used by the parser
```

Comment:
Can't this be added to the existing `is_tag_valid_with_parent` function?

Reply:
Well that is to check the single parent above...I can move it but to me it feels more right here

Reply:
Ah gotcha.